Lib/asyncio/timeouts.py

```diff
@@ -25,18 +25,36 @@ class _State(enum.Enum):
 
 @final
 class Timeout:
+    """Asynchronous context manager which limits the time spent inside it.
+
+    Example use:
+
+        loop = asyncio.asyncio.get_running_loop()
+        async with asyncio.Timeout(loop.time() + 1):  # Run for up to a second
+            await asyncio.sleep(2)                    # `TimeoutError` occurs.
+
+    """
 
     def __init__(self, when: Optional[float]) -> None:
+        """Initialize a timeout that will trigger at a given loop time.
```

Comment:
```suggestion
        """Schedule a timeout that will trigger at a given loop time.
```

Reply:
So this constructor also probably should state that it is not directly to be used. Or leave it out? The key things that need to be documented that currently aren't are the regular methods, i.e. when, reschedule, expired.